buildroot-external/package/rx8130-rtc/Config.in

```diff
@@ -0,0 +1,2 @@
+config BR2_PACKAGE_RX8130_RTC
+	bool "Support for Epson RX8130CE real time clock"
```

Comment:
The mainline kernel module does not support to enable the charging mode for supercaps as it is needed on the RPI-RF-MOD.

Reply:
This seems to be not entirely the case anymore (see https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/rtc/rtc-ds1307.c#n1045). However, I don't know since which kernel version the mainline module integrated the charging mode of the rx8130 in `ds1307.c`. I will have to check and test it and potentially switch to the mainline module, if possible. But there might easily be other reasons why we would have to keep up using this legacy driver. I will report as soon as I have tested it.

Reply:
@agners I just verified that the `rtc-ds1307` kernel module in recent 5.10.x kernels has the necessary functionality to charge the supercap used on the `RPI-RF-MOD`. Thanks for the hint! 

Thus, as per your suggestion, I removed the dedicated `rtc-rx8130` legacy kernel module and applied changes to the proposed device tree overlays of this PR. Thus, IMHO your requested changes should be fulfilled. Next will be to try to get the integration of `generic_raw_uart` more "generic" as per request of @alexreinert.